cmd/stakercli/pop/pop_test.go

```diff
@@ -0,0 +1,48 @@
+package pop_test
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/babylonlabs-io/btc-staker/itest/testutil"
+	"github.com/babylonlabs-io/btc-staker/staker"
+)
+
+var popsToVerify = []staker.Response{
+	{
+		BabyAddress:   "bbn1xjz8fs9vkmefdqaxan5kv2d09vmwzru7jhy424",
```

Comment:
just a question, as I assume this data was generated by generate pop command ?

Reply:
Ideally yes, but that would require some e2e tests to set up wallet, generate btc account, etc.